### PR TITLE
Update fastlane to include CRFHeartRate

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -21,7 +21,8 @@ platform :ios do
     ENV["MATCH_KEYCHAIN_NAME"] = "ios-build.keychain"
     ENV["TEAM_SAGEBIO"] = "4B822CZK9N"
     ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"] = "KA9Z8R6M6K"
-    ENV["APP_ID"] = "org.sagebase.CRFTestApp"
+    ENV["APP_ID_TEST"] = "org.sagebase.CRFTestApp"
+    ENV["APP_ID_HEARTRATE"] = "org.sagebase.CRFTestApp"
```
Comment (zaro0508) [inline fastlane/Fastfile]: shouldn't this be `org.sagebase.CRFHeartRate` ?

File: travis/build.sh [hunk 2]
```diff
@@ -7,9 +7,11 @@ elif [[ -z "$TRAVIS_TAG" && "$TRAVIS_BRANCH" == "master" ]]; then  # non-tag com
     echo "Build on merge to master"
     FASTLANE_EXPLICIT_OPEN_SIMULATOR=2 bundle exec fastlane test project:"CardiorespiratoryFitness/CardiorespiratoryFitness.xcodeproj"  scheme:"CRFTestApp"
     bundle exec fastlane ci_archive scheme:"CRFTestApp" export_method:"app-store" project:"CardiorespiratoryFitness/CardiorespiratoryFitness.xcodeproj"
+    bundle exec fastlane ci_archive scheme:"CRFHeartRate" export_method:"app-store" project:"CRFHeartRate/CRFHeartRate.xcodeproj"
```
Comment (zaro0508) [inline travis/build.sh]: don't you want this to to run tests for CRFHeartRate scheme when there's a PR?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: fastlane/Fastfile (hunk 1 above)
Comment (syoung-smallwisdom) [reply fastlane/Fastfile]: Whoops, thanks!

File: travis/build.sh (hunk 2 above)
Comment (syoung-smallwisdom) [reply travis/build.sh]: CRFHeartRate doesn't have any tests set up for it. It's inclusion is just for the duration of the validation study.

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**